src/pattern-library/routes.ts

```diff
@@ -53,7 +53,7 @@ export type PlaygroundRoute = {
    * not provided, a placeholder entry for this route will be added to the
    * navigation, with no link.
    */
-  route?: RegExp | string;
+  route?: string;
```

Comment:
Adjusted to work with `wouter` types. We only used `RegExp` in one route, which isn't needed anymore.